robottelo/ui/repository.py

```diff
@@ -138,6 +138,48 @@ def validate_field(self, name, field_name, expected_field_value):
                 'repo.fetch_' + field_name]).text == expected_field_value)
         return False
 
+    def fetch_content_count(self, repo_name, content_type):
+        """Fetch content count for specific repo and content type.
+
+        :param repo_name: Name of repository that should be fetched.
+        :param content_type: Type of repository content. Supported values are:
+
+            * packages
+            * errata
+            * package_groups
+            * puppet
+
+        :returns: Content count.
+        :rtype: int
+        :raises: ``ValueError`` if passed ``content_type`` is not supported.
+        :raises robottelo.ui.base.UINoSuchElementError: If content is not
+            found.
+        """
+        self.search_and_click(repo_name)
+        if content_type == 'packages':
```

Comment:
perhaps add a constant (like REPO_TYPE) that contains: packages, errata, package_groups, puppet, and from the tests use that constant